lib/presentation/widgets/registration_form.dart

```diff
@@ -57,21 +59,9 @@ class _MyLogFormWidgetState extends State<RegisterForm> {
           spacer,
           CustomButton(
             onPressed: () async {
-              try {
```

Comment:
WHY THE FUCK IS IT HERE????
NEED TO DISCUSS

Reply:
I deleted async and try